mf4_csv/main.py

```diff
@@ -0,0 +1,52 @@
+
+import os 
+from asammdf import MDF, Signal
+from os import walk
+import re
+import glob
+
+
+output_dir = "./output_in_csv"
+
+def prepare_dir(mf4_file):
+    my_str = str(mf4_file)
+    my_str = my_str.lower().strip(".mf4")
```

Comment:
What would happen if my filename had some uppercase letters? Will this code still function?

Reply:
Sure! although that ll be tested asap.